src/Widgets/NavigationWidget/SubComponents/CurrentUserDropdown.tsx

```diff
@@ -67,22 +67,32 @@ export const CurrentUserDropdown = connect(function CurrentUserDropdown({
         </>
       ) : null}
 
-      <NavDropdown.Item
-        as={LinkTag}
-        eventKey="MetaNavigation-LogOut"
-        key="MetaNavigation-LogOut"
-        onClick={() => {
-          logout()
-        }}
-        to={Obj.root()}
-      >
-        <i className={`bi bi-box-arrow-right`}></i>
-        {widget.get('logOutLabel')}
-      </NavDropdown.Item>
+      <LogOutButton logOutLabel={widget.get('logOutLabel')} />
     </NavDropdown>
   )
 })
 
+const LogOutButton = connect(function LogOutButton({
+  logOutLabel,
+}: {
+  logOutLabel: string
+}) {
+  return (
+    <NavDropdown.Item
+      as={LinkTag}
+      eventKey="MetaNavigation-LogOut"
+      key="MetaNavigation-LogOut"
+      onClick={() => {
+        logout()
+      }}
```

Comment:
FYI This can be simplified AFAICS

```suggestion
      onClick={logout}
```

Reply:
Actually TypeScript will complain:
```
Type '(returnTo?: string | undefined) => void' is not assignable to type 'MouseEventHandler<HTMLElement>'.
  Types of parameters 'returnTo' and 'event' are incompatible.
    Type 'MouseEvent<HTMLElement, MouseEvent>' is not assignable to type 'string'.
```